docker/entrypoint.sh

```diff
@@ -23,6 +23,8 @@ case "$1" in
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--workspace" "--" "--include-ignored"
         elif [[ "$TEST_FULL_SYNC" -eq "1" ]]; then
             exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--test" "acceptance" "--" "--nocapture" "--ignored" "full_sync_mainnet"
+        elif [[ "$TEST_LWD_RPC_SYNC" -eq "1" ]]; then
+            exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--" "--nocapture" "--ignored" "fully_synced_rpc_test"
```

Comment:
Performance: Let's only build and run the acceptance test binary here:
```suggestion
            exec cargo "test" "--locked" "--release" "--features" "enable-sentry" "--test" "acceptance" "--" "--nocapture" "--ignored" "fully_synced_rpc_test"
```